deploy/examples/cluster-test.yaml

```diff
@@ -19,7 +19,7 @@ data:
     mon_warn_on_pool_no_redundancy = false
     bdev_flock_retry = 20
     bluefs_buffered_io = false
-    mon_data_avail_warn = 500M
+    mon_data_avail_warn = 10
```

Comment:
So this needs to be set as in the recent CI has low availability in disk size,
So it always gives us a warning.
BY default the `mon_data_avail_warn` is set to 30%, so making it 10% so no such warning we should see in the CI.

Reply:
TL;DR I'd support removing the value entirely.

The default is a percentage (30%).

As a user new to ceph, when I hit this issue I quickly found that the expected value was 30%, so I didn't not understand why it was warning when I had 92% free. I didn't know to look for this config.

As a user new to ceph, I think it would have been least confusing to use the default value. If it was warning when I was actually low on disk space I would have been ok with that, even in a test environment.

Reply:
i agree to above points, we should not change default configuration in test yamls which user uses to deploy test clusters.

Reply:
Okay will discuss this and update you,
As of now, it is becoming lesser than that default value which won't give you any such warnings in the future,